### PR TITLE
Update docs now offers are not only EXPERIMENTAL_FEATURES

### PR DESCRIPTION
**Pull request by rustyrussell**
Changelog-None

### CODE REVIEW

File: doc/lightningd-config.5.md [hunk 1]
```diff
@@ -463,6 +463,25 @@ if C-lightning terminates.
 Built-in plugins, which are installed with lightningd(8), are automatically
 considered important.
 
+### Experimental Options
+
+Experimental options are subject to breakage between releases: they
+are made available for advanced users who want to test proposed
+features.  If lightningd is built with `--enable-experimental-features`
```
Comment (niftynei) [inline doc/lightningd-config.5.md:470]: Might not be totally obvious what "is built with" means from the outset; might be good to give people a clue via `configured`?

```
If lightningd is built configured with ...
```

**Review by niftynei — APPROVED** [review]
ACK 2c4de0708909195e663e7d6e545d3655098cef7d

**Comment by cdecker** [comment]
Clean rebase on top of `master` after merging #4330 and #4332

**Comment by cdecker** [comment]
Added missing `doc/lightning-sendonionmessage.7` file which was failing in `NO_PYTHON` configs.

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/4331/commits/b350c52ff360ac263bf08e73a9fed282b3a31d47

**Pull request merged**